Lib/profiling/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Python profiling tools.
+
+This package provides two types of profilers:
+
+- profiling.tracing: Deterministic tracing profiler that instruments every
+  function call and return. Higher overhead but provides exact call counts
+  and timing.
+
+- profiling.sampling: Statistical sampling profiler that periodically samples
+  the call stack. Low overhead and suitable for production use.
+"""
+
+__all__ = ["tracing", "sampling"]
```

Comment:
We don't need mutability here:
```suggestion
__all__ = ("tracing", "sampling")
```